_src/membership.adoc

```diff
@@ -0,0 +1,16 @@
+= How to join as a member
+
+Follow these steps:
+
+. Start by *creating an account on link:{open-collective}[Open Collective^]*,
+  if you don't have one. This is where we manage funds and pay
+  expenses.
+. *link:/register/[Register] as an affiliate* - if you've not done that
+  already (see xref:#affiliation[above].)
```

Comment:
```suggestion
. *link:/register/[Register] as an member* - if you've not done that
  already (see xref:#affiliation[above].)
```

I know we're holding off on releasing this page for right now, but I noticed a mistake here.
I think this should actually say register as a member, not an affiliate.

Reply:
I think the meaning I meant was just follow the instructions to register as per an affiliate, but I see your point.  

However I see this raises a problem: people who register as an affiliate first don't have a way to change their registered status to member currently. Someone with write access to the registrations would have to o that manually. We need to make our infrastructure a bit smarter.